frontend/src/components/SimpleBar/SimpleBar.js

```diff
@@ -1,6 +1,9 @@
 import Box from '@mui/material/Box'
```

Comment:
Similar to the bit about rounded corners in SimpleRange, SimpleBar's corners aren't round in Safari. That can be fixed by applying the same `borderRadius: '4px'` to LinearProgress-bar.